### ISSUE TITLE
[Suggestion] Walk Module To Confuse

### ISSUE THREAD

**Issue by IIpho3nix** [C10udburst/meteor-rejects#4]
Walks Around The Player In Confus Instead Of Just Teleporting

**Comment by IIpho3nix** [comment]
(If It Could Work witf Speed It Would Be Good)

**Comment by Cypphi** [comment]
> Walks Around The Player In Confuse Instead Of Just Teleporting

Make this a separate module called TargetStrafe. This would circle around the player from a set distance going from 0 to [insert amount here]. There should be an option to only work while killaura is enabled as well as an option to only work when speed is enabled. This would be useful to stay out of reach of a player as well as to circle around them to **confuse** them as well as make it harder for them to hit you.

**Basically Sigma TargetStrafe**

**Also maybe make it render a circle with the exact range you set it to for extra drip**

**Comment by mathmakgakpak** [comment]
circle in minecraft :WTF:

**Comment by GoofyBoi420** [comment]
use setting circle
